GitUI/Avatars/ICachedAvatarProvider.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace GitUI.Avatars
+{
+    /// <summary>
+    /// An <see cref="IAvatarProvider"/> with support for clearing avatar caches.
+    /// </summary>
+    public interface ICachedAvatarProvider : IAvatarProvider
+    {
+        /// <summary>
+        /// Raised when images cached by this implementation (if any)
+        /// </summary>
+        event Action CacheCleared;
```

Comment:
The doc and the API name contradict one another.

Reply:
I guess that was a typo in the original implementation. My english is not that great but I guess the new documentation should do the trick ^^
✔ done.